joint_trajectory_controller/test/joint_trajectory_controller_wrapping_test.cpp

```diff
@@ -0,0 +1,279 @@
+///////////////////////////////////////////////////////////////////////////////
+// Copyright (C) 2013, PAL Robotics S.L.
```

Comment:
Who do you want this copyright attributed to? I'll also adjust the year to 2017 unless you disagree.